app/source/css/components/PageMenu.styl

```diff
@@ -37,19 +37,18 @@
           font-size: 15px;
         }
 
-          &:hover {
-             border-bottom: 4px solid white;
-             padding-bottom: 14px !important;
-           }
-             
-           &.active {
-             a {
-               border-bottom: 4px solid;
-               padding-bottom: 14px !important;
-               outline: none;
-             }  
-           } 
-      } 
+        hover() {
+          border-bottom: 4px solid white;
+          padding-bottom: 14px !important;
```

Comment:
I completely agree. Feel free to remove ones that you see as clearly unneeded; in case of doubt, let us know and we can review (may be edge cases).